ni_measurementlink_service/_internal/discovery_client.py

```diff
@@ -329,7 +329,7 @@ def _start_service(
         try:
             with _open_key_file(str(key_file_path)) as _:
                 return discovery_service_subprocess
-        except IOError:
+        except OSError:
```

Comment:
What was the scenario where we were getting 'IOError' before and how are we handling this case now? Because it looks like we're no longer catching 'IOError'.

Reply:
https://docs.python.org/3/library/exceptions.html#ZeroDivisionError

> The following exceptions are kept for compatibility with previous versions; starting from Python 3.3, they are aliases of [OSError](https://docs.python.org/3/library/exceptions.html#OSError).
> 
> - exception EnvironmentError
> - exception IOError
> - exception WindowsError: Only available on Windows.